eclass/linux-mod-r1.eclass

```diff
@@ -279,6 +283,13 @@ fi
 # to split into two packages than use this variable due to requiring
 # rebuilds every kernel upgrades.
 
+# @ECLASS_VARIABLE: LINUX_MOD_R1_NO_KERNEL_DEPS
+# @PRE_INHERIT
+# @DEFAULT_UNSET
+# @DESCRIPTION:
+# If set to a non-null value no dependencies on virtual/linux-sources
+# or virtual/dist-kernel will be set by the eclass.
```

Comment:
Maybe adding intend of the variable wouldn't hurt, e.g.
`Note: intended to avoid circular dependencies when used by a kernel`
Don't want people adding this in random ebuilds.